lms/djangoapps/grades/scores.py

```diff
@@ -216,7 +216,7 @@ def _get_explicit_graded(block):
     return True if field_value is None else field_value
 
 
-def _weighted_score(raw_earned, raw_possible, weight):
+def weighted_score(raw_earned, raw_possible, weight):
```

Comment:
Nit: Please move this function to the top of the page since it's no longer an internal function.